src/client/Engine/Caching.jsx

```diff
@@ -35,30 +35,45 @@ const Caching = ({ onChange, caching }) => (
         />
       </Col>
     </Row>
+    <Row>
+      <Col md={2}>
+        <OIbCheckBox
+          label={caching.archive.enabled ? 'Archive Mode activated' : 'Archive Mode desactivated'}
+          name="engine.caching.archive.enabled"
+          value={caching.archive.enabled}
+          defaultValue
+          help={<div> Move to Archive Folders or Delete files</div>}
+          onChange={onChange}
+          switchButton
+        />
+      </Col>
+    </Row>
+    {caching.archive.enabled && (
     <Row>
       <Col md={4}>
         <OIbText
-          name="engine.caching.archiveFolder"
+          name="engine.caching.archive.archiveFolder"
           label="Archive Folder"
-          value={caching.archiveFolder}
-          defaultValue="./cache/archive"
-          valid={validation.engine.caching.archiveFolder}
+          value={caching.archive.archiveFolder}
+          defaultValue="./cache/archived"
+          valid={validation.engine.caching.archive.archiveFolder}
           help={<div>Where to store the cached data</div>}
           onChange={onChange}
         />
       </Col>
       <Col md={2}>
-        <OIbSelect
-          label="Archive Mode"
-          name="engine.caching.archiveMode"
-          options={['archive', 'delete']}
-          value={caching.archiveMode}
-          defaultValue="archive"
-          help={<div> Move to Archive Folders or Delete files</div>}
+        <OIbInteger
+          name="engine.caching.archive.retentionDuration"
+          label="Retention Duration"
+          value={caching.archive.retentionDuration}
+          defaultValue={8760}
```

Comment:
Thinking back at this value, I think one year is too big. Can you make it to one month (30 days) instead ?